File: 1.py
import math

p = 1
for i in range(1,11):
# 000
    s1 =0
    s2 =0
    for i in range(1,p+1):
        s1 += math.sqrt(i)
        s2 += 1/math.sqrt(i)

    s = s1*s2/p/p 
    p=p*10
    print(str(p)+" " +str(s) )

File: 1.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    float s1=0,s2=0;
    long long x;
    cin>>x;
    for(int i=1;i<=x;i++){
        s1+=sqrt(x);
        s2+=1/sqrt(x);

    }
    float res=s1*s2/pow(x,2);
    cout<<res<<endl;
}